src/Icon/Icon.d.ts

```diff
@@ -0,0 +1,18 @@
+import { CSSProperties, ComponentClass, MouseEventHandler } from 'react';
+import { Type } from 'tcomb';
+
+export type IconProps = {
+  icon?: string,
+  color?: string,
+  paths?: number, // it was PositiveInteger
+  onClick?: MouseEventHandler<HTMLElement>,
```

Comment:
I checked: there is no `HTMLIElement`. internally react uses generic `HTMLElement` for `<i>`